apollo-federation/src/merge.rs

```diff
@@ -1378,6 +1530,49 @@ fn join_graph_enum_type(
     (join_graph_enum_name, join_graph_enum_type)
 }
 
+fn add_core_feature_inaccessible(supergraph: &mut Schema) {
+    // @link(url: "https://specs.apollo.dev/inaccessible/v0.2")
+    supergraph
+        .schema_definition
+        .make_mut()
+        .directives
+        .push(Component::new(Directive {
+            name: name!("link"),
+            arguments: vec![
+                Node::new(Argument {
+                    name: name!("url"),
+                    value: "https://specs.apollo.dev/inaccessible/v0.2".into(),
```

Comment:
Small nit: you could use the identity for inaccessible here to not have this hard-coded. Something like:

```rust
let spec = InaccessibleSpecDefinition::new(Version {major: 0, minor: 2});
...
    value: spec.to_string(),
```